resources/dicts/thoughts/alive/kitten.json

```diff
@@ -26,28 +26,28 @@
     {
         "id": "gen_kit",
         "thoughts": [
-            "Plays moss ball by themselves",
-            "Wonders who their mentor will be",
+            "Plays moss ball by {PRONOUN/m_c/subject}",
+            "Wonders who {PRONOUN/m_c/subject} mentor will be",
```

Comment:
looks like a lot of the subject pronouns you've put in should be poss pronouns.  'their' is a possessive, not a subject